mantidproject/mantid
# Plot difference checkbox added to muon guis

**Description of work.**
Added a plot difference checkbox to muon guis that displays or removes plot difference curve when clicked


**To test:**
1. Open muon analysis gui and load appropriate data.
2. Go to fitting tab add a fitting function and click on fit button.
3. Click on "Plot Difference" checkbox on top right of plotting window then click on fit. Plot difference curve should now appear.
4. Open frequency domain analysis gui and load appropriate data.
5. go to transform tab and click on calculate FTT.
6. Repeat steps 2 and 3.



Fixes #29070 . 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
